projects/arlas-toolkit/src/lib/tools/utils.ts

```diff
@@ -332,8 +333,30 @@ export function hasContributorData(contributor: Contributor): boolean {
   }
 }
 
-export function orderAlphabeticallyArlasSearchFields(arlasSearchFieldA: ArlasSearchField, arlasSearchFieldB: ArlasSearchField,){
+export function orderAlphabeticallyArlasSearchFields(arlasSearchFieldA: ArlasSearchField, arlasSearchFieldB: ArlasSearchField,) {
   const comparedWordA = arlasSearchFieldA?.label.trim().toLowerCase();
   const comparedWordB = arlasSearchFieldB?.label.trim().toLowerCase();
   return comparedWordA.localeCompare(comparedWordB);
 }
+
+export function flattenData(y: Object) {
```

Comment:
If we want to be able to use it elsewhere more, I think we could add as an optional parameter the character used to flatten, in case we need to use another one